robottelo/hosts.py

```diff
@@ -257,6 +257,24 @@ def subscription_manager_status(self):
     def subscription_manager_list(self):
         return self.execute('subscription-manager list')
 
+    def subscription_manager_get_pool(self, sub_list=[]):
+        pool_ids = []
+        for sub in sub_list:
+            result = self.execute(
+                f'subscription-manager list --available --pool-only --matches="{sub}"'
+            )
+            result = result.stdout
+            result = result.split('\n')
+            result = ' '.join(result).split()
```

Comment:
Can be combined into one:
```suggestion
            result = result.stdout
            result = result.split('\n')
            result = ' '.join(result.stdout.split('\n')).split()
```
Non blocking Comment!

Reply:
I tried to be a little explicit for better readability. 
I will have a followup PR for all the small improvements.